docs/MIGRATING.md

```diff
@@ -45,6 +45,9 @@
 - `SubMsg` and `Reply` now have a new `Payload` field. This contains arbitrary
   bytes from the contract that should be passed through to the corresponding
   `Reply` call.
+- If you link statically against `wasmvm_muslc`, you should not rename the
+  downloaded library anymore, as wasmvm now directly links against
+  `wasmvm_muslc.x86_64` / `wasmvm_muslc.aarch64` instead of `wasmvm_muslc`.
```

Comment:
Since the project is doing the linking internally [through these Go files](https://github.com/CosmWasm/wasmvm/blob/v2.0.0-rc.2/internal/api/link_muslc_aarch64.go#L5), let's rephase to the file names. I.e. you need to provide `wasmvm_muslc.x86_64.a` / `wasmvm_muslc.aarch64.a` instead of `wasmvm_muslc.a` and the Go build system does the linking for you.